stable/xray/values.yaml

```diff
@@ -43,8 +105,8 @@ xray:
     shared:
       logging:
         consoleLog:
-          enabled: {{ .Values.xray.consoleLog }}
-      jfrogUrl: "{{ tpl (required "\n\nxray.jfrogUrl is required! This allows to connect to Artifactory.\n(You can copy the JFrog URL from Admin > Security > Settings)" .Values.xray.jfrogUrl) . }}"
+          enabled: {{ include "xray.consoleLog" . }}
```

Comment:
I think this should be false by default for all helm charts. There's no value for it in k8s. It just duplicates the Docker's stdout.

Reply:
By default `xray.consoleLog` is set to false in values.yaml .